package.json

```diff
@@ -6,81 +6,77 @@
   "engines": {
     "node": ">=18 <21"
   },
-  "main": "index.js",
-  "prettier": "eslint-config-vinta/prettier",
```

Comment:
I dropped the custom prettier config, because it's not compatible yet with latest prettier version... cc @victorfsf 
Also, is it worth maintaining that? I guess it's too difficult to keep this up-to-date.

Reply:
Actually, Rangel had already updated this! https://github.com/vintasoftware/eslint-config-vinta/pull/14
So I will try again.